### ISSUE TITLE
Request your provider (new/planned/complete)

### ISSUE THREAD

**Issue by Unode** [SirCmpwn/pass-rotate#9]
In order to organize work on which providers to add next and to compile a list of all providers planned/targeted, this thread will serve as a compilation list:

In alphabetical order:
[?] - Amazon
[ ] - Bing
[x] - Cloudflare
[x] - DigitalOcean
[x] - Discord
[ ] - Disqus
[ ] - Docker
[x] - Github
[x] - Google
[x] - Github
[x] - Linode
[x] - Namecheap
[x] - Pixiv
[ ] - Steam
[x] - Twitter
[ ] - Yahoo
[x] - Y Combinator

**Comment by SirCmpwn** [comment]
The Amazon provider is partial, see #7. I also spent some time on PayPal support with limited success.

**Comment by lelutin** [comment]
hey there. Gandi could be a nice addition. also they have an API that could probably be used for this.

it would be nice also to see some providers for some more generic free software that one could point to any domain where they host them. some examaples could be redmine, wordpress, drupal

**Comment by Unode** [comment]
@SirCmpwn Feel free to edit the post as you see fit.

@lelutin Added.

**Comment by SirCmpwn** [comment]
I agree with the generic idea. If there are no domain-specific providers, I want to detect if it supports the standard (#1) and then detect if it uses something open source (and if it does, use i.e. MediaWikiProvider).

**Comment by evaryont** [comment]
Howdy, this is just a few that jump out to me, going through my password manager:

- Mozilla
- Newsblur
- LastPass, Bitwarden, etc
- ownCloud, Nextcloud
- Reddit
- Lobste.rs
- Pinboard
- Dominos
- Papa John's
- Mojang/Minecraft
- Twitch
- HumbleBundle
- Nintendo ID (I think this is separate from Club Nintendo, Pokemon. I don't own a switch, hopefully it's just Nintendo ID too)
- Microsoft (Skype, too? But they are trying to convert all skype accounts into Microsoft accounts)
- Slack

Is the amazon provider solely for the online shopping component? Or will it cover AWS as well? (AWS root accounts as well as IAM credentials will need to be considered.)

**Comment by SirCmpwn** [comment]
The current Amazon provider (that does not work) only covers Amazon. Not sure how AWS is different.

**Comment by Immortalin** [comment]
outlook.com

**Comment by np511** [comment]
itch.io would be nice

**Comment by gabm** [comment]
I added GitLab... see #30

**Comment by GrayHatter** [comment]
sr.ht

**Comment by leo60228** [comment]
I'd say that Nintendo ID isn't *overly* important, it has SSO with Google and IIRC Facebook.
